src/rgw/rgw_sal_motr.cc

```diff
@@ -2649,6 +2664,89 @@ int MotrAtomicWriter::process(bufferlist&& data, uint64_t offset)
   return this->write();
 }
 
+int MotrObject::update_null_version_index(const DoutPrefixProvider *dpp, rgw_bucket_dir_entry& ent)
+{
+  int rc, ret_code = -ENOENT;
```

Comment:
init ret_code to 0 only
also use intuitive var names, including for rc & ret_code here

Reply:
sure 
removed `ret_code = -ENOENT`
done.